readme.md

```diff
@@ -3,15 +3,15 @@
 # Skipper
 
 Skipper is an HTTP router that acts as a reverse proxy (with support for flexible route definitions), alters
-requests, and responses with filters. It:
+requests, and responds with filters. It:
```

Comment:
Sure. The confusion from my edit stems from the original use of a comma that changes the meaning of your intent (I changed "responses" to "responds," to be grammatically consistent). If you use this new intro, here's how it should go (minus the edit, which is a question):

Skipper is an HTTP router that acts as a reverse proxy, mapping incoming requests to multiple HTTP backend services based on routes selected by the request attributes. In between [EDIT: in between what and what?], both the requests and responses can be augmented by a filter chain defined individually for each route.